### DISCUSSION TITLE
How to convert a PDB file of an antibody to AHO scheme?

### DISCUSSION THREAD

**Discussion by XHHnefu** [RosettaCommons/rosetta#214] [PyRosetta]
Hi,

RosettaAntibodyDesign tutorial mentions 'The input PDB file must be renumbered to the AHo Scheme and the light chain gene must be identified. Thiscan be done through the PyIgClassify Server'.

But I don't see any submit options or tools. There is only Search, Statistic and Download options.(http://dunbrack2.fccc.edu/PyIgClassify/)(http://dunbrack2.fccc.edu/PyIgClassify2/default.aspx)

One researcher mentioned that 'The submit page converts PDBs (http://dunbrack2.fccc.edu/PyIgClassify/User/UserPdb.aspx)'. But this URL indicates that the resource cannot be found.

**Comment by yikzlim** [comment]
Hi, I'm facing the problem to renumber my Ab PDB file to AHo scheme too. Have you found any alternative to do the Ab numbering?

**Comment by roccomoretti** [comment]
Additional discussion regarding antibody renumbering in Discussion #297
